lib/std/posix.zig

```diff
@@ -5624,52 +5626,52 @@ pub fn dl_iterate_phdr(
 
 pub const ClockGetTimeError = error{UnsupportedClock} || UnexpectedError;
 
-/// TODO: change this to return the timespec as a return value
-pub fn clock_gettime(clock_id: clockid_t, tp: *timespec) ClockGetTimeError!void {
+pub fn clock_gettime(clk_id: clockid_t) ClockGetTimeError!timespec {
```

Comment:
I guess the rename from `clock_id` to `clk_id` can be reverted throughout.

Reply:
yep